### DISCUSSION TITLE
Issue adding HACS as Home Assistant integration

### DISCUSSION THREAD

**Discussion by randalljraitzii** [home-assistant/architecture#555] [Documentation] [answered]
After clicking "Add Integration" and searching for HACS, a window appears with 4 check boxes but no text. After some research, I've found that these are likely consent agreements for usage (i.e. I know how to access logs, I know there are no add-ons in HACS, I know that everything inside HACS is custom..., I know that if I get issues with Home Assistant I should disable all my custom_components.

No matter what I do, check all boxes and click Submit, I just get a spinning status indicator and I'm never prompted to input my github access token.

This is on a fresh install of Home Assistant 2021.4.6 using the install command `wget -q -O - https://install.hacs.xyz | bash -`

I'm ready to blow it all away and start over again, but figured I'd ask if anyone has come up against this and knows a quick solution?
![hacs_accept](https://user-images.githubusercontent.com/60020203/116011555-df270380-a5e2-11eb-9ad0-c29fd953440c.png)

**Comment by randalljraitzii** [comment]
FWIW: I wiped my SD card and started from scratch and it worked. I believe that I'm being caught up in a bunch of documentation errors. The writeup I was following stated I would be able to enter a github token, but after accepting the consents, I was required to perform device based auth against github, which I don't think this screen ever came up my first installation attempt.

**Reply by spacegaier** [reply]
I think GitHub recently changed their token approach and now a device token is required. The official HACS docs however seem to reflect that already correctly.

Either way not an architecture topic at all. For HACS support, see https://hacs.xyz/help.

**Comment by frenck** [comment, marked as answer]
This discussion is misplaced. This repository is for discussing architectural changes/issues with Home Assistant, however, you are raising an issue for HACS, which is not managed by the Home Assistant project.

Locking down this issue for that reason.
